pkg/util/throttle.go

```diff
@@ -26,9 +26,12 @@ type RateLimiter interface {
 	Accept()
 	// Stop stops the rate limiter, subsequent calls to CanAccept will return false
 	Stop()
+	// Saturation returns a percentage number which describes how saturated
+	// this rate limiter is.
```

Comment:
I'd suggest a slightly different semantics: there are 2 different 'full' levels, all burst tokens are gone, and we're using tokens in the same rate we're producing them (i.e. we're at full capacity of 'normal' rate). I'd rather have 1.0 mean the latter, and values above 1.0 the former (concrete value depends on how much given RL can burst)